### ISSUE TITLE
Error loading page

### ISSUE THREAD

**Issue by Tempusfugit65** [rvolz/BicBucStriim#237]
Hi! From one day to the other my library isn't working anymore. I did some updates from Synology (PHP, Apache and so on) and bang.... Error loading page if I try to login or opening the settings!! Installation check fails with "Open_basedir" warning. Strangely my library folder is not recognized anymore... Because I thought the updates caused the problem I uninstalled everything from my Synology NAS and installed one version below of PHP and Apache. Unfortunately the problem persists. Please help me...

![screenshot 52](https://cloud.githubusercontent.com/assets/25087092/21904349/92ec0d36-d903-11e6-901b-4dc4c91e2b79.png)

![screenshot 49](https://cloud.githubusercontent.com/assets/25087092/21904121/ddc4a846-d902-11e6-992c-129a6f062717.png)

![screenshot 51](https://cloud.githubusercontent.com/assets/25087092/21904279/6086a14e-d903-11e6-91cd-90fcf4465f9f.png)

![screenshot 50](https://cloud.githubusercontent.com/assets/25087092/21904278/60807b98-d903-11e6-9d5e-e6e425c16f59.png)

[2017-01-12.txt](https://github.com/rvolz/BicBucStriim/files/702613/2017-01-12.txt)

Thanks!!

**Comment by Tempusfugit65** [comment]
Can anyone help me please?

**Comment by rvolz** [comment]
Maybe the updates changed the access rights on your library folder? If everything else is unchanged, I would look into that. The server (http user) must be able to read that folder and its contents.

The diagnostics below the red light on your screenshot show that the folder either doesn't exist or the web server has no access rights for it.

**Comment by simistef** [comment]
I am experiencing the exact same issue.

DS214play
Current DSM version:
DSM 6.1.4-15217 Update 5
DSM date:
2017/12/20

BibBucStream 1.3.6-5


Yellow in installcheck only for open_basedir, and Red in directory not found.

Any steps that we can debug this, please.

**Comment by rvolz** [comment]
I guess you solved your problem, since you posted issue #270. If so, maybe you could help out the original poster?

**Comment by simistef** [comment]
Yes. I don't know how I solved it... The only thing I did was to install COPS package which worked and then BBS as well.

**Comment by DoctorDri** [comment]
I know this thread is old, but I found something interesting about it that solved it. I was having the same issue: after installing BBS, I couldn't login. Everytime I entered the default user and password (admin/admin) it returned the error "Error loading page". So it was impossible to configure BBS nor logging in.

I read several threads about this topic to no avail, until I read the last response here, where they said that installing the package COPS solved it. I did it, and while it wasn't COPS what fixed it, it did solve the problem.

What happens is that COPS requires PHP 5.6, so it's mandatory to install it before COPS. I did it and, once I selected PHP 5.6 on WebStation general settings, BBS started working and now I can log-in. Issue solved.
